packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts

```diff
@@ -24,17 +24,17 @@ const iconStyle = {
 
 /**
  * Gets the base button styles. Note: because it is a base class to be used with the `mergeRules`
- * helper, it should have values for all class names in the interface. This let `mergeRules` optimize
+ * helper, it should have values for all class names in the interface. This const `mergeRules` optimize
```

Comment:
How did you make the changes? search/replace or using tslint autofix? We shouldn't change comments

Reply:
Thanx for the review @cschleiden. This and a few others are already fixed.